### ISSUE TITLE
Doesn't work on 2024.4.0b0

### ISSUE THREAD

**Issue by tbrasser** [agittins/bermuda#147]
## Version of the custom_component

Latest

## Configuration

Worked on latest on 2024.3.x

## Describe the bug

![Screenshot_2024-03-28-11-28-27-122_io.homeassistant.companion.android.jpg](https://github.com/agittins/bermuda/assets/2816560/31739206-f254-4e46-96fb-7dde54fbdf1e)

## Debug log

```text

Add your logs here.

```

**Comment by sambarlick** [comment]
I'm also on the beta and having the same issue, even after the upgrade to 0.6.1.

This one in my logs... Logger: homeassistant.config_entries
Source: config_entries.py:2589
First occurred: 07:55:15 (56 occurrences)
Last logged: 09:20:50

Error occurred loading flow for integration bermuda: Platform bermuda.config_flow not found


Also I have attached my debug logs.
[error_log-6.txt](https://github.com/agittins/bermuda/files/14817802/error_log-6.txt)

**Comment by agittins** [comment]
Thanks both for the heads-up! I don't track the betas myself so it's extremely helpful that other folk are testing this out.
Thanks @sambarlick for including your debug logs, that helps a lot so I know what to aim for when trying to reproduce.

I'm currently updating my test environment (which includes a python version bump) so hopefully I'll be able to reproduce and solve this soon.

Interesting that floors looks and home maps appear to be a new feature in this release, that might have some significant impacts for this project!

**Comment by agittins** [comment]
I wasn't able to reproduce the issue directly (although the same issue happened for the `google_translate` core component on first boot, but not on subsequent boots). However there are a few fixes in `config_flow.py` that ensure it works OK on 2024.4.0b4 that I've pushed in #149 

If you can choose the "main" branch in HACs and test that would be appreciated! (Go into HACS, choose Bermuda, in the top-right meatballs menu choose "Redownload" and select "main" as the version).

**Comment by tbrasser** [comment]
Works a charm!

**Comment by agittins** [comment]
Great!

Just a heads-up, I just noticed that the private_ble_device integration changed how it named its device trackers in 2024.4, which caused Bermuda to create new sensors that will show up under a separate device, instead of in the shared private_ble_device "device". I just pushed PR #151 which fixes that. The downside is that after updating you'll have to delete the old devices manually to clean up, sorry.

Note this only affects those who have installed `main` between v0.6.1 and `now`.
